pkg/ibmvpc/examples/out/analysis_out/lb_bad_practice_all_vpcs_.txt

```diff
@@ -4,7 +4,7 @@ alb[LoadBalancer] => vsi0-sub1[10.240.0.4] : All Connections**
 alb[LoadBalancer] => vsi0-sub2[10.240.64.4] : All Connections**
 alb[LoadBalancer] => vsi0-sub3[10.240.128.4] : All Connections**
 vsi0-ctrl-sub[10.240.2.4] => Public Internet (all ranges) : All Connections
-vsi0-ctrl-sub[10.240.2.4] => alb[LoadBalancer] : All Connections**
+vsi0-ctrl-sub[10.240.2.4] => alb[LoadBalancer] : All Connections ***
```

Comment:
its a combination of the below. I agree it is confusing and should be fixed, but not in this PR. This PR revealed this because the LB abstraction prior to this PR ignores the stateful property. 
We should probably open an issue for V0.5, the same release and the other relevant PRs that causes this ***
```

connections are stateful (on TCP) unless marked with *
connections marked with ** are an over-approximation, not all private IPs have the same connectivity
```

Reply:
at least add a space after the first `*` for now?

Reply:
done, we should consider further improving - in which case we should open a followup issue